code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -317,13 +317,18 @@
 			H.verbs |= verb_path
 	return
 
-/datum/species/proc/handle_post_spawn(var/mob/living/carbon/human/H) //Handles anything not already covered by basic species assignment.
+/datum/species/proc/handle_post_spawn(var/mob/living/carbon/human/H,var/kpg) //Handles anything not already covered by basic species assignment. Keepgene value should only be used by genetics.
```

Comment:
Please explicitly define a default value instead of depending on BYOND defaulting this to `null`.